keras_nlp/src/models/mobilenet_v3/mobilenet_v3_backbone_test.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+
+from keras_nlp.src.models.mobilenet_v3.mobilenet_v3_backbone import (
+    MobileNetV3Backbone,
+)
+from keras_nlp.src.tests.test_case import TestCase
+
+
+class MobileNetV3BackboneTest(TestCase):
+    def setUp(self):
+        self.init_kwargs = {
+            "stackwise_expansion": [
```

Comment:
Try to make much smaller test models. It will be a lot more readable, and most importantly keep our test suite faster.

In general, try to hit the interesting forms of config variation (different filters, kernels, activations, etc), but with as small constants as possible. `"stackwise_filters": [4, 8, 16]`, etc